Header header
uint8 payload_len
uint8 seq
uint8 sys_id
uint8 comp_id
uint32 msg_id
uint8[] payload
uint16 checksum

